siva/checkpoint.go

```diff
@@ -0,0 +1,169 @@
+package siva
+
+import (
+	"os"
+	"strconv"
+
+	borges "github.com/src-d/go-borges"
+	billy "gopkg.in/src-d/go-billy.v4"
+	"gopkg.in/src-d/go-billy.v4/util"
+	errors "gopkg.in/src-d/go-errors.v1"
```

Comment:
remove the useless package allias

Reply:
These aliases are now added automatically by `goimports`:

https://github.com/golang/go/issues/28428